src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -237,6 +237,7 @@ protected override async Task<bool> AcceptEvent(SessionId sessionId, string meth
                         {
                             case var _ when url == "":
                             case var _ when url.StartsWith("wasm://", StringComparison.Ordinal):
```

Comment:
is there a reason we're using`url.StartsWith("wasm://"` here and `url.StartsWith("wasm://wasm/",` in SendCallStack?

Reply:
Done.